lib/lambda_wrap/lambda_manager.rb

```diff
@@ -84,8 +84,13 @@ def deploy_lambda(
 
       begin
         @client.get_function(function_name: function_name)
-        func_config = @client.update_function_code(function_name: function_name, s3_bucket: bucket, s3_key: key,
-                                                   s3_object_version: version_id, publish: true).data
+        @client.update_function_code(function_name: function_name, s3_bucket: bucket, s3_key: key,
+                                     s3_object_version: version_id, publish: true).data
+        vpc_configuration = { subnet_ids: vpc_subnet_ids, security_group_ids: vpc_security_group_ids } unless (vpc_subnet_ids.empty? && vpc_security_group_ids.empty?)
+        func_config = @client.update_function_configuration(function_name: function_name, role: lambda_role, runtime: runtime,
```

Comment:
Can you output the result of the `update_function_code` separately? The individual `puts <result-from-previous-call>` are useful information if something went wrong during deployment.

The rest looks good.